packages/demo/src/styles/general-guidelines.tsx

```diff
@@ -116,44 +134,46 @@ export default () => (
                 </ul>
             </Section>
         </Section>
-        <Section title="Messages / Warning">
-            Warning messages are used to help users prevent error situations. They appear in a yellow banner at the top
-            of the screen and move the content below it. It should contain a suggested next step. It should be rejected
-            only by the user.
+        <Section level={1} title="Messages / Warning">
+            <p className="body-m-book-subdued">
+                Warning messages are used to help users prevent error situations. They appear in a yellow banner at the
+                top of the screen and move the content below it. It should contain a suggested next step. It should be
+                rejected only by the user.
+            </p>
         </Section>
-        <Section title="Inline validation">
-            <p>
+        <Section level={1} title="Inline validation">
+            <p className="body-m-book-subdued">
                 Inline Validation is direct feedback displayed as the user progresses through the form to confirm that
                 their answers are valid. Inline Validation helps people complete forms quickly and with less effort,
                 fewer errors, and more satisfaction. The opposite approche is to validate all the inputs at the end of
                 the process, when the user submits the form.
-            </p>
-            <p>
+                <br></br>

```

Comment:
I think having paragraphs was cleaner than having BRs

Reply:
i'll change this in my current branch :+1: